daffodil-core/src/main/scala/org/apache/daffodil/dsom/AnnotatedSchemaComponent.scala

```diff
@@ -400,7 +400,12 @@ trait AnnotatedMixin { self: AnnotatedSchemaComponent =>
       {
         ai.attribute("source") match {
           case None => {
-            this.SDW(WarnID.AppinfoNoSource, """xs:appinfo without source attribute. Is source="http://www.ogf.org/dfdl/" missing?""")
+            // if a child node in the dfdl namespace exists we will provide a warning about using the source property
+            ai.child.flatMap(n => Option(n.namespace)).find{_.contains("dfdl")}.foreach { _ =>
```

Comment:
In the ``case Some`` below, we use contains("ogf") || contains("dfdl") to consider a DFDL namespace "acceptable". Should we use the same logic here?

Reply:
done